node/src/main/scala/com/wavesplatform/transaction/assets/exchange/Order.scala

```diff
@@ -21,11 +23,11 @@ case class Order(
     matcherPublicKey: PublicKey,
     assetPair: AssetPair,
     orderType: OrderType,
-    amount: TxAmount,
-    price: TxAmount,
+    amount: TxExchangeAmount,
+    price: TxOrderPrice,
     timestamp: TxTimestamp,
     expiration: TxTimestamp,
-    matcherFee: TxAmount,
+    matcherFee: Long,
```

Comment:
Interesting. We leave this, because have a validation in `isValid`?

Reply:
Made it refined